docs/modules/community/pages/mailing-lists.adoc

```diff
@@ -0,0 +1,165 @@
+= Mailing lists
+
+== Guidelines
```

Comment:
How about listing mailing list first, then detailing the guidelines?

Reply:
Hmmm... We can definitely discuss about this. I know on the legacy documentation website the list of mailing lists comes first. But I was thinking... Does people are really gonna notice and read the guidelines if we put the list first?

I don't say putting the guidelines first will make everybody read them, but at least they will notice it, and probably more will read. They can always easily jump to the list after. 

Is this necessary or did I overthink a bit here? :)

Reply:
I think it delays the valuable information people look for. But I'm not really opinionated on this.